service/app/vdict_service/models.py

```diff
@@ -2,5 +2,5 @@
 
 
 class Word(models.Model):
-    name = models.TextField()
+    name = models.TextField(unique=True, max_length=100)
```

Comment:
MySQL can index only the first N chars of a BLOB or TEXT column. Since TextField doesn’t have a defined length, you can’t mark it as unique=True. MySQL will report: “BLOB/TEXT column ‘<db_column>’ used in key specification without a key length”. Related to this link:

[Textfield Limitations](https://docs.djangoproject.com/ja/1.9/ref/databases/#textfield-limitations)